02_Working_With_ASDF_Files/02-Working_With_ASDF_Files.ipynb

```diff
@@ -0,0 +1,382 @@
+{
```

Comment:
<p>A side comment about WCS probably is useful (e.g., this represents how to transform pixel coordinates to sky coordinates)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/asdf-format/scipy2022tutorial/pull/10/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Reply:
done